### PR TITLE
Misc fixes and cleanup

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# see:
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
+* @openziti/maintainers
```
Comment (andrewpmartinez) [inline .github/CODEOWNERS:3]: whooo hoo!

File: deps/libsodium.cmake [hunk 2]
```diff
@@ -23,9 +24,10 @@ if (WIN32)
            message("libsodium = ${libsodium_lib_path}")
     else()
            FetchContent_Declare (
-		   libsodium
-		   URL	https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable-mingw.tar.gz
-		   )
+                   libsodium
+                   URL	https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable-mingw.tar.gz
+                   URL_HASH MD5=110bbb6ca2322a0c2b3f639432e15cea
```
Comment (andrewpmartinez) [inline deps/libsodium.cmake:29]: Was this added because of a tool upgrade or because we specifically added it?

**Review by andrewpmartinez — APPROVED** [review]

File: deps/libsodium.cmake (hunk 2 above)
Comment (ekoby) [reply deps/libsodium.cmake:29]: I just added because it seemed like a good idea

File: .github/CODEOWNERS (hunk 1 above)
Comment (ekoby) [reply .github/CODEOWNERS:3]: maintainers still need to be poked to review PRs

**Pull request merged**